.github/workflows/container.yml

```diff
@@ -16,20 +16,22 @@ jobs:
       - name: Build the container
         run: docker build -t image-builder -f distribution/Dockerfile-ubi .
 
-      - name: Run the container
-        run: |
-          docker run -d --net=host --name image-builder image-builder
-          docker ps
-
-      - name: Verify that the startup log line is in the logs
-        run: |
-          docker logs image-builder | grep "Starting image-builder server"
-
-      - name: Check that the API is responding
-        run: curl -sv http://127.0.0.1:8086/api/image-builder/v1/version
-
-      - name: Stop the container
-        run: docker stop --time 1 image-builder
+      # - name: Run the container
```

Comment:
We don't have docker here, and image-builder needs a psql instance. So i'm unsure what to do here.

Maybe we need to change it in such a way that image-builder fails to start when there's no DB there, but just marks itself as 'not ready'. We need to add a readiness check/endpoint anyway and this might be a good use case. We could start imagebuilder, ask for the version, and make sure it's not ready?

Reply:
Doesn't github have service containers just for this kind of test?

Reply:
I did not know about this! i'll look into it

Reply:
This is what might interest you: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservices

Also, I don't know any details but I used `docker` and `podman` without any issues in GH Actions.